micrometer-core/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "org.jetbrains.kotlin.jvm" version "1.7.22"
+    alias(libs.plugins.kotlin17)
```

Comment:
We will continue using the same Kotlin plugin version. We don't want to upgrade the minor version in a maintenance release.